### ISSUE TITLE
Should loss be set to 0

### ISSUE THREAD

**Issue by ishaanthakur** [pytorch/tutorials#1290]
https://github.com/pytorch/tutorials/blob/48e5cccecbd385e00b95cb60ecdfecd06a43282b/intermediate_source/char_rnn_generation_tutorial.py#L281

Instead of having loss as zero should it instead be something like
loss = torch.autograd.variable(torch.Tensor([0]))
